connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/outbound/jobhandling/SpringConnectorJobHandler.java

```diff
@@ -71,6 +71,12 @@ public SpringConnectorJobHandler(
 
   @Override
   public void handle(JobClient client, ActivatedJob job) {
+    String jobTypeAndId =
+        job.getCustomHeaders().getOrDefault("id", "unknown")
+            + "#"
+            + job.getCustomHeaders().getOrDefault("version", "0");
+    metricsRecorder.increase(
+        Outbound.METRIC_CONNECTOR_VERSION, Outbound.JOB_RECEIVED, jobTypeAndId);
```

Comment:
As discussed the `jobType`, `elementTemplateId` & `elementTemplateVersion` should be kept separate and stored as individual tags.